scripts/bnsd/genesis_app_state.json

```diff
@@ -2,46 +2,29 @@
   "cash": [
     {
       "address": "b1ca7e78f74423ae01da3b51e676934d9105f282",
-      "coins": [
-        {
-          "whole": 123456789,
-          "ticker": "CASH"
-        }
-      ]
+      "coins": [ "123456789 CASH" ]
     },
     {
       "address": "053e4ffbb040d4dc3b201dfe9264c00d918e087b",
-      "coins": [
-        {
-          "whole": 1000000,
-          "ticker": "CASH"
-        },
-        {
-          "whole": 2000000,
-          "ticker": "BASH"
-        }
-      ]
+      "coins": [ "1000000 CASH", "2000000 BASH" ]
     }
   ],
   "currencies": [
     {
       "ticker": "CASH",
       "name": "Main token of this chain",
-      "sig_figs": 6
     },
     {
       "ticker": "BASH",
       "name": "Another token of this chain",
-      "sig_figs": 6
     },
     {
       "ticker": "ASH",
       "name": "Let the Phoenix arise",
-      "sig_figs": 6
```

Comment:
Could you run a `yarn prettier --write scripts/bnsd/genesis_app_state.json scripts/bcpd/genesis_app_state.json` to fix the broken JSON (trailing comma in the line above)?